webdriver/tests/support/fixtures.py

```diff
@@ -263,12 +263,19 @@ def create_dialog(dialog_type, text=None):
         assert isinstance(text, basestring), "`text` parameter must be a string"
 
         # Script completes itself when the user prompt has been opened.
+        # For prompt() dialogs, add a value for the 'default' argument,
+        # as some user agents (IE, for example) do not produce consistent
```

Comment:
Kudos for resisting "(i.e. IE)" :P

Reply:
@jugglinmike True confession time, I very nearly did "(e.g., IE)".